time-manager/System/TimeManager.hs

```diff
@@ -84,7 +86,7 @@ data State
 
 ----------------------------------------------------------------
 
--- | Creating timeout manager which works every N microseconds
+-- | Creating timeout manager which works every N micro seconds
```

Comment:
I think [`microsecond`](https://en.wikipedia.org/wiki/Microsecond) is one word in English. (I'm not an English native speaker either, btw, so 🤷 )

Reply:
Thanks.